HdProduction.HelpDesk.Api/Models/Tickets/CommentRequestModel.cs

```diff
@@ -0,0 +1,10 @@
+namespace HdProduction.HelpDesk.Api.Models.Tickets
+{
+    public class CommentRequestModel
```

Comment:
I think we also need CreatedAt property with the time of creation.

Reply:
agree, but not in request model, I'll add property to entity